README.md

```diff
@@ -1,5 +1,6 @@
 Authors Haven - A Social platform for the creative at heart.
 =======
+[![CircleCI](https://circleci.com/gh/andela/forsetti-ah-frontend.svg?style=svg)](https://circleci.com/gh/andela/forsetti-ah-frontend)
```

Comment:
Good job. Don't you think the badges should come at the top of the README.md file?. Instaed of placing it in between the repository description. It will make the README neater.

Reply:
Ok noted, made changes